docs/types/mapping-types.rst

```diff
@@ -25,7 +25,7 @@ in functions, or as parameters for library functions.
 They cannot be used as parameters or return parameters
 of contract functions that are publicly visible.
 
-You can mark variables of mapping type as ``public`` and Solidity creates a
+You can mark state variables of mapping type as ``public`` and Solidity creates a
```

Comment:
This is the only area that required clarification, so maybe worth double checking if this correction is true.